src/todo/components/TodoList.tsx

```diff
@@ -52,19 +98,28 @@ const ToDo = () => {
                     onChange={(e) => setDescription(e.target.value)}
                     placeholder="Enter task description"
                 />
-                </div>
+                <button type="submit">Add task</button>
+                </form>
+                <button onClick={removeAllTodos}>Remove All todos</button>
+                <button onClick={markAllTodosAsChecked}>Mark All Todos As Checked</button>
+                <button onClick={async()=>{
```

Comment:
better to move it to the handler func

Reply:
I don’t understand how to move them to the handle function?))